planner/core/task.go

```diff
@@ -554,7 +554,18 @@ func (p *PhysicalBroadCastJoin) GetCost(lCnt, rCnt float64) float64 {
 		rightSchema:   p.children[1].Schema(),
 	}
 	numPairs := helper.estimate()
+	if p.JoinType == SemiJoin || p.JoinType == AntiSemiJoin || p.JoinType == LeftOuterSemiJoin || p.JoinType == AntiLeftOuterSemiJoin {
```

Comment:
inner join also could have other conditions, like t1 join t2 on t1.a = t2.a and **t1.b>t2.b**

Reply:
The cost calculation model here for `PhysicalBroadCastJoin` is the same as `PhysicalHashJoin`.